UI/css/styles.css

```diff
@@ -151,6 +151,7 @@ nav {
 }
 /* landing page  */
 #landing-page {
+  padding: 6.4rem 0;
```

Comment:
Try to use percentage(hint 10%). padding: 10% 0;

Reply:
![image](https://user-images.githubusercontent.com/67368268/96427631-b3236380-11f6-11eb-8fe7-4cada3e4c3fc.png)
I think it is working perfectly, Thanks for the feedback.